### ISSUE TITLE
Mesh has invalid or extra triangles

### ISSUE THREAD

**Issue by el-dee** [Moguri/panda3d-gltf#33]
Testing with https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/MetalRoughSpheres loading the .gltf shows invalid or extra triangles:

![Screenshot from 2019-10-07 22-07-45](https://user-images.githubusercontent.com/44778133/66344888-25f4d000-e94f-11e9-9ed6-85ac3fa7a3d4.png)

The actual mesh is a pure array of spheres :

https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/MetalRoughSpheres/screenshot/screenshot.png

**Comment by el-dee** [comment]
I tried with half a dozen other glTF viewers (Babylon.js, Three.js, Hilo3D, ...) none displays those wrong triangles.

I investigated a bit by dumping the loaded vertex arrays and primitives but nothing obvious appears except that the bogus triangles are the lasts triangles of the last mesh. Also, they correspond to the label (Metalness, Roughness, ...) that should be displayed next to the spheres.

I also tested with other complex glTF files and some also displays distorted or missing mesh or part of the meshes. Simple glTF files do not display errors though, so there might be a memory corruption occurring somewhere.
